docker-compose-gpu.yml

```diff
@@ -46,4 +51,5 @@ services:
     environment:
       - API_ENDPOINT=http://haystack-api:8000
       - EVAL_FILE=eval_labels_example.csv
+      - HAYSTACK_UI_DISABLE_FILE_UPLOAD=1
```

Comment:
What we could do as an alternative is to just pass the environment variable from the system (ec2 instance) to the container. And set the environment variable in the system when it's automatically created.